server/api/v1/project_service.go

```diff
@@ -136,13 +147,11 @@ func (s *ProjectService) UpdateProject(ctx context.Context, request *v1pb.Update
 		}
 	}
 
-	projectMsg, err := s.store.UpdateProjectV2(ctx, patch)
+	project, err = s.store.UpdateProjectV2(ctx, patch)
 	if err != nil {
 		return nil, status.Errorf(codes.Internal, err.Error())
 	}
-	// TODO(d): create activity for project transfer.
```

Comment:
Is deleting this line is by design?

Reply:
It was the wrong place to comment. This should be done in UpdateDatabase().